Cargo.toml

```diff
@@ -322,6 +322,7 @@ xcm = { package = "staging-xcm", git = "https://github.com/moonbeam-foundation/p
 xcm-builder = { package = "staging-xcm-builder", git = "https://github.com/moonbeam-foundation/polkadot-sdk", branch = "moonbeam-polkadot-stable2409", default-features = false }
 xcm-executor = { package = "staging-xcm-executor", git = "https://github.com/moonbeam-foundation/polkadot-sdk", branch = "moonbeam-polkadot-stable2409", default-features = false }
 xcm-runtime-apis = { git = "https://github.com/moonbeam-foundation/polkadot-sdk", branch = "moonbeam-polkadot-stable2409", default-features = false }
+snowbridge-core = { git = "https://github.com/moonbeam-foundation/polkadot-sdk", branch = "moonbeam-polkadot-stable2409", default-features = false }
```

Comment:
It looks like this dependency is only added to use the `AllowSiblingsOnly` struct. I would suggest to re-implement the logic, without taking the dependency.

Reply:
I've reimplemented it.